app/pb_kits/playbook/pb_weekday_stacked/_weekday_stacked.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= content_tag(:div,
```

Comment:
Did you consider using a [Time tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)? It would provide a more semantic way of presenting users with this value, and provides a canonical way for providing a machine readable value.

Reply:
That's a great idea!